api/admin/routes.py

```diff
@@ -386,7 +386,9 @@ def admin_view(collection=None, book=None, **kwargs):
                     quoted_book,
                     quoted_book.replace("/", "%2F"))
             return redirect(app.manager.url_for('admin_sign_in', redirect=redirect_url))
-        library = Library.instance(app.manager._db)
+        # TODO: Design the admin interface for multiple libraries.
+        # Until then, pick some library so the admin interface can run.
+        library = app.manager._db.query(Library).all()[0]
```

Comment:
Can you sort this by Library.id? Since we'll be reintroducing the 'default library' concept soon it's OK if we wait and solve it there, but that's a small change.